deps.edn

```diff
@@ -6,6 +6,7 @@
         net.cgrand/macrovich {:mvn/version "0.2.1"}
         net.cgrand/xforms {:mvn/version "0.19.2"}
         io.github.inferenceql/inferenceql.inference {:git/sha "40e77dedf680b7936ce988b66186a86f5c4db6a5"}
+        io.github.inferenceql/inferenceql.gpm.sppl {:git/sha "f745dbb0c17c1a9da21488b7bd3098338ab7d7a2"}
```

Comment:
I think there is an optimization here which I anticipate including in a change (forthcoming) that moves to more centralized and reuable Nix code in `inferenceql/nix`. unfortunately, since `clj-nix#mk-deps-cache` doesn't support selecting an alias, we are obliged to do a little extra chicanery to manage multiple lockfiles based on conditions. Figuring that out will make more sense when the dimensionality of our cross-org nix usage is clearer.

It would be more urgent even now if `inferenceql.gpm.sppl`'s clojure dependencies were more severe, or brought along python stuff automagically, which it does not.

So you're right that overall there is a loose end here that will be worn down one way or another.